libraries/AP_HAL_ChibiOS/hwdef/MountainEagleH743/hwdef.dat

```diff
@@ -0,0 +1,195 @@
+# hw definition file for processing by chibios_pins.py
+# for MountainEagleH743
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1444
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+FLASH_SIZE_KB 2048
+env OPTIMIZE -Os
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 128
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 12
+define CH_CFG_ST_RESOLUTION 16
+
+# USB
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# STM32-Debug-Interface
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+# SPI1 for IMU1 (BMI270) as a second IMU
+PC15 IMU1_CS     CS
+PA5  SPI1_SCK    SPI1
+PA6  SPI1_MISO   SPI1
+PD7  SPI1_MOSI   SPI1
+
+# SPI2 for AT7456 OSD
+PB12  AT7456_CS  CS
+PB13  SPI2_SCK   SPI2
+PB14  SPI2_MISO  SPI2
+PB15  SPI2_MOSI  SPI2
+
+# SPI3 - external
+PD4   EXT_CS     CS
+PB3   SPI3_SCK   SPI3
+PB4   SPI3_MISO  SPI3
+PB5   SPI3_MOSI  SPI3
+
+# SPI4 for IMU2 (ICM42688)
+PE11  IMU2_CS    CS
+PE12  SPI4_SCK   SPI4
+PE13  SPI4_MISO  SPI4
+PE14  SPI4_MOSI  SPI4
+
+# two I2C bus
+I2C_ORDER I2C2 I2C1
+
+# I2C1
+PB6  I2C1_SCL  I2C1
+PB7  I2C1_SDA  I2C1
+
+# I2C2
+PB10 I2C2_SCL  I2C2
+PB11 I2C2_SDA  I2C2
+
+# ADC
+PC0  BATT_VOLTAGE_SENS  ADC1  SCALE(1)
+PC5  BATT_CURRENT_SENS  ADC1  SCALE(1)
+
+define HAL_BATT_MONITOR_DEFAULT  4
+define HAL_BATT_VOLT_PIN        10
+define HAL_BATT_CURR_PIN        11
+define HAL_BATT_VOLT_SCALE      11.0
+define HAL_BATT_CURR_SCALE      40.0
+
+PC1 RSSI_ADC ADC1
+define BOARD_RSSI_ANA_PIN 8
+
+# LED
+# LED0-orange
+# LED1-green
+PE9 LED0 OUTPUT LOW GPIO(90) # orange
```

Comment:
in addition to get it to compile, you will need to add this new define to get the bd leds to work```suggestion
define AP_NOTIFY_GPIO_LED_2_ENABLED 1
PE9 LED0 OUTPUT LOW GPIO(90) # orange
```